#Service used to send velocity-radius information to the ArcDriveController

#Tangential Velocity To Maintain
float64 velocity

#Radius of arc to create
float64 radius
